eclipse-che/che-operator
# Deploy Issuer and Certificate for the webhook when deploying DWO on Kubernetes

### What does this PR do?
`$TITLE`

### Screenshot/screencast of this PR
N/A


### What issues does this PR fix or reference?
eclipse/che#19944

### How to test this PR?
1. Use minikube or other Kubernetes cluster.
1. Make sure `cert-manager` is installed in the cluster.
1. Deploy `che-operator` built from this branch (there is a test image `quay.io/lkrejci/che-operator:isue-19944` with the code as was at the time of creation of this PR). 
1. Enable devworkspaces in the `CheCluster` CR.
1. Check that deploying a devfile v2 works correctly.


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
